R/FilterStateChoices.R

```diff
@@ -204,6 +210,11 @@ ChoicesFilterState <- R6::R6Class( # nolint
 
       private$set_choices_counts(unname(table(x_factor)))
 
+      if (private$multiple) {
+        private$init_null <- FALSE
+      } else {
+        private$init_null <- TRUE
+      }
```

Comment:
The variable private$init_null is set to TRUE when private$multiple is set to FALSE. This is because in the observeEvent for input$selection , the radio button, something has to be selected for input$selection, and it cannot be NULL. However, in the checkbox, it can be NULL. Hence, the variable private$init_null is used.